docs/cudf/source/developer_guide/testing.md

```diff
@@ -218,6 +218,29 @@ This way, when the bug is fixed, the test suite will fail at this
 point (and we will remember to update the test).
 
 
+### Testing code that throws warnings
+
+Some code may be expected to throw warnings.
+A common example is when a cudf API is deprecated for future removal, but many other possibilities exist as well.
+The cudf testing suite [surfaces all warnings as errors](https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#controlling-warnings).
```

Comment:
Can we use “latest” or “stable” links instead of pytest 7.1.x links?

Reply:
Done.